snuba/migrations/policies.py

```diff
@@ -83,3 +83,24 @@ def can_run(self, migration_key: MigrationKey) -> bool:
 
     def can_reverse(self, migration_key: MigrationKey) -> bool:
         return True
+
+
+MIGRATION_WEIGHTS = {
```

Comment:
Is there a better way to do this? Just wanted an easy way to compare policies

Reply:
This comparison strategy seems a little bit fragile. If we add more policies it seems like it may become unwieldy keeping a total order of policies. Also, I believe you would have to ensure the comparator is transitive for python to work properly and this can be hard to figure out for some permissions. 

A better way to do this would be dump all the policies into a list for each user, then the checker enumerates through and allows if it finds one that has the permissions. Its a little slower, but given that the size of these lists should be very small, the perf impact is negligible.

Reply:
I agree with Dalitso, this seems really complicated.

```
# Sorted by importance
MIGRATION_WEIGHTS = ["All", "Non", "No"]

def max_policy(policies: Sequence[str]) -> str:
    for p in MIGRATION_WEIGHTS:
        if p in policies:
            return p
```

Reply:
update this re dalitso's suggestion, and now just check all the policies for a group simplifying this so that there is no need for weights